### ISSUE TITLE
gevent.pool.Group.imap not equivalent to itertools.imap

### ISSUE THREAD

**Issue by thomasst** [gevent/gevent#565]
`gevent.pool.Group.imap` suggests that it's equivalent to `itertools.imap`. However, `itertools.imap` accepts multiple iterables (`itertools.imap(function, *iterables)`), whereas `gevent.pool.Group.imap` only supports one:

```
    def imap(self, func, iterable):
        """An equivalent of itertools.imap()"""
```

Either the documentation should be clarified and say that it's *not* equivalent to `itertools.imap()`, or (better) the method should be changed to be equivalent to `itertools.imap` and accept multiple iterables.

**Comment by thedrow** [comment]
Could we use ```itertools.chain()```? If we can then it's a simple change.

**Comment by jamadden** [comment]
Fixed via ddb0bccce41a6ec62503191a395a1de1863bb20b
